src/libs/TextInputUtils/index.native.js

```diff
@@ -0,0 +1,10 @@
+function getPasswordAutocompleteType() {
+    return 'password';
+}
+
+function setNativePropsWeb() {}
```

Comment:
One reason why we move things into `index.js` and `index.native.js` files is so we don't have to have code that mentions a specific platform. Can we name this something more generic and also add some explanation about why we are not running this code on native?

Reply:
Ok. I will update this.
Mostly, the reason is that I am using this to set the web attributes needed for browser perspective. (name, for, ...etc).